build.gradle

```diff
@@ -41,64 +41,56 @@ idea {
 dependencies {
 
     //Versions managed by Spring Boot Dependency Management
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind'
-    compile group: 'com.fasterxml.jackson.core', name: 'jackson-core'
-    compile group: 'commons-codec', name: 'commons-codec'
-    compile group: 'org.hibernate', name: 'hibernate-entitymanager'
-    compile group: 'org.hibernate', name: 'hibernate-core'
-    compile group: 'javax.servlet', name: 'jstl'
-    compile group: 'org.apache.commons', name: 'commons-lang3'
-    compile group: 'org.apache.commons', name: 'commons-pool2'
-    compile group: 'org.apache.httpcomponents', name: 'httpclient'
-    compile group: 'org.flywaydb', name: 'flyway-core'
-    compile group: 'org.postgresql', name: 'postgresql'
-    compile group: 'org.springframework.boot', name: 'spring-boot'
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-web'
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter'
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-jdbc'
-    compile group: 'org.springframework.boot', name: 'spring-boot-autoconfigure'
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa'
-    compile group: 'org.xerial', name: 'sqlite-jdbc'
-    compile group: 'redis.clients', name: 'jedis'
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind'
```

Comment:
Is this group name change required by the gradle upgrade?

Reply:
Not quite, but soon. `compile` is headed for deprecation in version 7.